src/brain/issueLabelHandler/index.test.ts

```diff
@@ -738,7 +760,7 @@ describe('issueLabelHandler', function () {
     it('should not modify labels when community member comments and issue is waiting for product owner', async function () {
       await setupIssue();
       await addLabel(WAITING_FOR_PRODUCT_OWNER_LABEL, 'routing-repo');
-      await addComment('routing-repo', 'Picard');
```

Comment:
noticed this test case had the wrong user. `Picard` is an owner, not a community member